pkgs/development/tools/minizinc/default.nix

```diff
@@ -1,9 +1,9 @@
-{ lib, stdenv, fetchFromGitHub, cmake, flex, bison }:
+{ lib, stdenv, fetchFromGitHub, callPackage, cmake, flex, bison, gecode, mpfr, cbc, zlib }:
 stdenv.mkDerivation rec {
```

Comment:
Since this file gets edited, I think it would be a good opportunity to use the `finalAttrs` pattern here.

Let me share a couple of useful links regarding this:

1. History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
2. Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
3. Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

Feel free to come back to me if you need further help.

Cheers.

Reply:
Oh, this is cool to know about, thanks!